loan-calculator-client/src/components/AdminSignup.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Card, Col, Row, Input, Button } from 'antd';
+import firebase from '../scripts/firebase.js'
+
+
+
+class AdminSignup extends React.Component {
+	constructor(props) {
+		super(props)
+		this.state = {
+			email: "",
+			password: ""
+		}
+		this.handleChange = this.handleChange.bind(this)
+	}
+	render() {
+		return (
+			<div>
+				<Row gutter={10} style={{paddingTop: '10px'}}>
+					<Col span={10} style={{margin: 'auto'}}>
+						<Card title="Create An Account!" bordered={true}>
+						Email: <Input placeholder="john@regions.com" name='email' value={this.state.email} onChange={this.handleChange}/>
+						<br />
+						<br />
+						Password: <Input.Password placeholder="password123" name='password' value={this.state.password} onChange={this.handleChange}/>
```

Comment:
prob shouldn't be suggesting a password but it's pretty funny

Reply:
I am suggesting a password, so people are aware of good password habits. I changed the placeholder to: p@$$w0r6. This is also something we can say that we added for our security hardening requirement for the project.